Example/Analytics/DemoAppAnalytics.swift

```diff
@@ -74,16 +74,17 @@ public class DemoAppAnalytics {
     }
 
     internal class func getComponentName(cell: UITableViewCell) -> String? {
-        guard let miniAppCell = cell as? MiniAppCell else {
-            guard let customPermissionCell = cell as? CustomPermissionCell else {
-                guard let customPermissionCell = cell as? FirstLaunchCustomPermissionCell else {
-                    return cell.textLabel?.text
-                }
-                return customPermissionCell.permissionTitle.text
-            }
-            return customPermissionCell.titleLabel?.text
-        }
-        return miniAppCell.titleLabel?.text
+//        guard let miniAppCell = cell as? MiniAppCell else {
+//            guard let customPermissionCell = cell as? CustomPermissionCell else {
+//                guard let customPermissionCell = cell as? FirstLaunchCustomPermissionCell else {
+//                    return cell.textLabel?.text
+//                }
+//                return customPermissionCell.permissionTitle.text
+//            }
+//            return customPermissionCell.titleLabel?.text
+//        }
+//        return miniAppCell.titleLabel?.text
+        return nil
```

Comment:
Do we have Demo app analytics maintained in new changes ?

Reply:
as discussed I'll address it in a new PR